store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1650,25 +1663,49 @@ private boolean isSpaceToDelete() {
             cleanImmediately = false;
 
             {
-                double physicRatio = UtilAll.getDiskPartitionSpaceUsedPercent(getStorePathPhysic());
-                if (physicRatio > diskSpaceWarningLevelRatio) {
+                String[] storePaths;
+                String commitLogStorePath = DefaultMessageStore.this.getMessageStoreConfig().getStorePathCommitLog();
+                if (commitLogStorePath.contains(MessageStoreConfig.MULTI_PATH_SPLITTER)) {
+                    storePaths = commitLogStorePath.trim().split(MessageStoreConfig.MULTI_PATH_SPLITTER);
+                } else {
+                    storePaths = new String[]{commitLogStorePath};
```

Comment:
not necessary if branch, could use the same code with MULTI_PATH branch

Reply:
updated.